CandidateReranking.md

```diff
@@ -0,0 +1,369 @@
+Candidate Reranking Service V1
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+    - [Query](#query)
+        - [Query](#query)
+        - [ParsedInput](#parsed-input)
+        - [ParsedEntity](#parsedentity)
+        - [Keyword](#keyword)
+        - [Relationship](#relationship)
+        - [Entity](#entity)
+        - [Filter](#filter)
+    - [Profile](#profile)
+        - [Profile](#profile)
+        - [RecentJob](#recentjob)
+        - [Skill](#skill)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+### Summary
+The Candidate Reranking Service is an HTTP Rest service which performs machine learned ranking on a list of candidate profiles. The service makes use of a versioned reranking-library jar maintained by Textkernel which applies a model trained on Talent Discovery Platform search data.
+
+### Request Structure
+The service supports POST requests to:
+https://api.careerbuilder.com/core/search/candidatereranker.
+
+OAUTH credentials are **required**.
+
+A request is composed of 4 main parts:
+`reranker_config`, `source`, `query`, and `profiles`.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| reranker_config | String | **TRUE** | Accepted Value: **RERANKER_V1** The `reranker_config` determines which learned model to apply to your incoming rerank request. Currently there is only one reranker model to call, "RERANKER_V1".
+| source | String | **TRUE** | Accepted Values: **EDGE** or **MY_SUPPLY**. The source identifies where profile data was attained. A model can be learned based on the source providing more accurate results per each data format.
```

Comment:
Should be 'obtained' not 'attained'.

Reply:
Also probably need a comma after "source" in the second sentence. Perhaps better as "A model can be learned based on the source, thereby providing more accurate results per each data format."